tests/io/sample_udp_inout/client.py

```diff
@@ -0,0 +1,83 @@
+import socket
+import select
+import time
+import logging
+import struct
+import sharpy.io.message_interface as message_interface
+import numpy as np
+"""
+This is not a test but is to be used as client when testing the development of the input
+output capabilities of sharpy.
+
+Run this script as client.
```

Comment:
I do not understand this "test". I am supposed to run it in two different computers?

Reply:
yes... that was the idea. Mainly because I wouldn't know how to run two processes in parallel in Travis so that's why I left it that way, as a test but one that requires the user to run it on their own. one terminal for the server and another for the client

Reply:
I think this is good enough. You do not want to mess up with travis